xDrip/ApplicationSupport/AppDelegate.swift

```diff
@@ -49,3 +51,35 @@ import UIKit
     }
     #endif
 }
+
+// MARK: - UNUserNotificationCenterDelegate
+extension AppDelegate: UNUserNotificationCenterDelegate {
```

Comment:
Still not done. Create a separate worker/controller, which would be `UNUserNotificationCenterDelegate` (instead of `AppDelegate`).